### PR TITLE
chore(package): eslint-config-airbnb 8.0.0

### PR DESCRIPTION
**Pull request by etaoins**
- Update eslint-config-airbnb to 8.0.0

- Lots of trivial fixes for new rules

- Ignore the bower_components/ directory

- Fix sass-lint at 1.5.1 as the newer versions appear to choke on our SASS

### CODE REVIEW

**Comment by davepoon** [comment]
LGTM :thumbsup:

File: src/helpers/classSuffixHelper.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ import _ from 'lodash';
 const classSuffixHelper = ({ classSuffixes, suffixOptions, componentClass }) => {
   const internalSuffixes = _.compact(classSuffixes);
 
-  _.forEach(suffixOptions, (value, optionName) => { if (value) {internalSuffixes.push(_.kebabCase(optionName));} });
+  _.forEach(suffixOptions, (value, optionName) => { if (value) internalSuffixes.push(_.kebabCase(optionName)); });
```
Comment (antklim) [inline src/helpers/classSuffixHelper.js]: `{` and `}` are not necessary here as well

**Comment by antklim** [comment]
LGTM 💯

File: src/helpers/classSuffixHelper.js (hunk 1 above)
Comment (etaoins) [inline src/helpers/classSuffixHelper.js]: This is performing a side effects. I want to keep the braces.

**Comment by etaoins** [comment]
@davepoon @antklim Merge?

**Pull request merged**